README.md

```diff
@@ -170,6 +170,51 @@ Before actually uploading anything to iTunes, ```deliver``` will generate a HTML
 no, en-US, en-CA, fi, ru, zh-Hans, nl-NL, zh-Hant, en-AU, id, de-DE, sv, ko, ms, pt-BR, el, es-ES, it, fr-CA, es-MX, pt-PT, vi, th, ja, fr-FR, da, tr, en-GB
 ```
 
+## Default values
+
+Deliver has a special ```default``` language code which allows you to provide values that are not localised, and which will be used as defaults when you don’t provide a specific localised value.
+
+You can use this either in json within your deliverfile, or as a folder in your metadata file.
+
+Imagine that you have localised data for the following language codes:  ```en-US, de-DE, el, it```
+
+You can set the following in your deliverfile
+
+```
+release_notes({
```

Comment:
what am I missing? There is a missing ' after the default, but this works, and I thought it was the correct syntax for a hash in ruby?
It's also the syntax you use for description in deliverfile.md

````
Localised values should be set like this

description({
  'en-US' => "English Description here",
  'de-DE' => "Deutsche Beschreibung hier"
})
```